Makefile

```diff
@@ -2123,13 +2115,13 @@ baremetal_cleanup: ## deletes the operator, but does not cleanup the service res
 
 ##@ DATAPLANE
 .PHONY: dataplane_prep
```

Comment:
We probably don't need these  targets as dataplane kuttl tests already use `openstack` target.